doc/src/manual/environment-variables.md

```diff
@@ -0,0 +1,317 @@
+larger values correspond to larger amounts of time.
+
+Suppose the value of `$JULIA_PKGRESOLVE_ACCURACY` is `n`. Then
+
+*   the number of pre-decimation iterations is `20*n`,
+*   the number of iterations between decimation steps is `10*n`, and
+*   at decimation steps, at most one in every `20*n` packages is decimated.
+
+## External applications
+
+### `JULIA_SHELL`
+
+The absolute path of the shell with which Julia should execute external
+commands (via `Base.repl_cmd()`). Defaults to the environment variable
+`$SHELL`, and falls back to `/bin/sh` if `$SHELL` is unset.
+
+### `JULIA_EDITOR`
+
+The editor returned by `Base.editor()` and used in, e.g., [`Base.edit`](@ref).
+
+`$JULIA_EDITOR` takes precedence over `$VISUAL`, which in turn takes precedence
+over `$EDITOR`. If none of these environment variables is set, then the editor
+is taken to be `open` on Windows and OS X, or `/etc/alternatives/editor` if it
+exists, or `emacs` otherwise.
+
+!!! note
+
+    `$JULIA_EDITOR` is *not* used in the determination of the editor for
+    [`Base.Pkg.edit`](@ref): this function checks `$VISUAL` and `$EDITOR`
+    alone.
+
+## Parallelization
+
+### `JULIA_CPU_CORES`
+
+Overrides the global variable [`Base.Sys.CPU_CORES`](@ref), the number of
+logical CPU cores available.
+
+### `JULIA_WORKER_TIMEOUT`
+
+A `Float64` that sets the value of `Base.worker_timeout()` (default: `60.0`).
+This function gives the number of a seconds a worker process will wait for
+a master process to establish a connection before dying.
+
+### `JULIA_NUM_THREADS`
+
+An unsigned 64-bit integer (`uint64_t`) that sets the maximum number of threads
+available to Julia. If `$JULIA_NUM_THREADS` exceeds the number of available
+physical CPU cores, then the number of threads is set to the number of cores.
+If `$JULIA_NUM_THREADS` is not positive or is not set, or if the number of CPU
+cores cannot be determined through system calls, then the number of threads is
+set to `1`.
+
+### `JULIA_THREAD_SLEEP_THRESHOLD`
+
+If set to a string that starts with the case-insensitive substring
+`"infinite"`, then spinning threads never sleep. Otherwise,
+`$JULIA_THREAD_SLEEP_THRESHOLD` is interpreted as an unsigned 64-bit integer
+(`uint64_t`) and gives, in nanoseconds, the amount of time after which spinning
+threads should sleep.
+
+### `JULIA_EXCLUSIVE`
+
+If set to anything besides `0`, then Julia's thread policy is consistent with
+running on a dedicated machine: the master thread is on proc 0, and threads are
+affinitized. Otherwise, Julia lets the operating system handle thread policy.
+
+## REPL formatting
+
+Environment variables that determine how REPL output should be formatted at the
+terminal. Generally, these variables should be set to [ANSI terminal escape
+sequences](http://ascii-table.com/ansi-escape-sequences.php). The Julia base
+package provides a high-level interface with much of the same functionality:
```

Comment:
"base package" isn't a term we use much, I guess you just mean functions that are under Base?